index.html

```diff
@@ -53,42 +54,51 @@ <h2>Kitchen Sink:</h2>
           <rh-footer-social-link slot="social-links-end" icon="web-icon-github"><a aria-label="Github"
               href="#github">Github</a>
           </rh-footer-social-link>
-          <rh-footer-links slot="links">
-            <h3 slot="header">Products</h3>
-            <rh-footer-link><a href="#">Red Hat Ansible Automation Platform</a>
-            </rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat Enterprise Linux</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenShift</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenShift Container Storage</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenStack Platform</a></rh-footer-link>
-            <rh-footer-link><a href="#">See all products</a></rh-footer-link>
-          </rh-footer-links>
-          <rh-footer-links slot="links">
-            <h3 slot="header">Tools</h3>
-            <rh-footer-link><a href="#">My account</a></rh-footer-link>
-            <rh-footer-link><a href="#">Customer support</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenShift</a></rh-footer-link>
-            <rh-footer-link><a href="#">Contact training</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenStack Platform</a></rh-footer-link>
-            <rh-footer-link><a href="#">See all products</a></rh-footer-link>
-          </rh-footer-links>
-          <rh-footer-links slot="links">
-            <h3 slot="header">Try, buy, sell</h3>
-            <rh-footer-link><a href="#">Red Hat Store</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat Enterprise Linux</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenShift</a></rh-footer-link>
-            <rh-footer-link><a href="#">Contact training</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat OpenStack Platform</a></rh-footer-link>
-            <rh-footer-link><a href="#">See all products</a></rh-footer-link>
-          </rh-footer-links>
-          <rh-footer-links slot="links">
-            <h3 id="communicate" slot="header">Communicate</h3>
-            <rh-footer-link><a href="#">Contact us</a></rh-footer-link>
-            <rh-footer-link><a href="#">Feedback</a></rh-footer-link>
-            <rh-footer-link><a href="#">Social</a></rh-footer-link>
-            <rh-footer-link><a href="#">Red Hat newsletter</a></rh-footer-link>
-            <rh-footer-link><a href="#">Email preferences</a></rh-footer-link>
-          </rh-footer-links>
+          <rh-footer-links-wrapper slot="links">
```

Comment:
I'm not a huge fan of `<rh-footer-links-wrapper><rh-footer-links>`. It strikes me as though we're exposing private APIs. If it's possible to just have `<rh-footer-links>`, that's preferable

Reply:
I'm not either, but it was the easiest way to keep the existing flexbox styles.  I can take another pass at it.

Reply:
This was removed.